changelog/unreleased/pull-2821

```diff
@@ -0,0 +1,6 @@
+cmd/mount: honour --no-lock flag
+
+Do not lock the repository if --no-lock global flag is set. This allows
+to mount repositories which are archived on a read only system.
+
+https://github.com/restic/restic/pull/2821
```

Comment:
Please add https://github.com/restic/restic/issues/1597 below this line.

Reply:
Done in 02c2ba9.